examples/pre-commit/basic/README.md

```diff
@@ -0,0 +1,54 @@
+# Basic Pre-commit Example
```

Comment:
This feels like a lot of documentation -- which will eventually get out of date -- to maintain. Can we just point people at the precommit hooks docs?

Reply:
Fair point, I'll remove these and replace with a higher level section in the docs. Also modify the tests to use an existing example policy.